controllers/admin/AdminTranslationsController.php

```diff
@@ -181,18 +181,19 @@ public function initToolbar()
     public function initMain()
     {
         // Block add/update a language
-        $packs_to_install = array();
-        $packs_to_update = array();
+        $packsToInstall = array();
+        $packsToUpdate = array();
         $token = Tools::getAdminToken('AdminLanguages'.(int)Tab::getIdFromClassName('AdminLanguages').(int)$this->context->employee->id);
-        $file_name = $this->link_lang_pack.'?version='._PS_VERSION_;
-        $array_stream_context = @stream_context_create(array('http' => array('method' => 'GET', 'timeout' => 8)));
-        if ($lang_packs = Tools::file_get_contents($file_name, false, $array_stream_context)) {
-            if ($lang_packs != '' && $lang_packs = json_decode($lang_packs, true)) {
-                foreach ($lang_packs as $key => $lang_pack) {
-                    if (!Language::isInstalled($lang_pack['iso_code'])) {
-                        $packs_to_install[$key] = $lang_pack;
+        $arrayStreamContext = @stream_context_create(array('http' => array('method' => 'GET', 'timeout' => 8)));
+        
+        if ($langPacks = Tools::file_get_contents($this->link_lang_pack, false, $arrayStreamContext)) {
+            if ($langPacks != '' && $langPacks = json_decode($langPacks, true)) {
+                foreach ($langPacks as $locale => $langName) {
```

Comment:
Yes, with the second parameters of `json_decode` set to `true` it's an associative array.
http://fr.php.net/manual/en/function.json-decode.php